rollup.config.js

```diff
@@ -0,0 +1,18 @@
+import terser from '@rollup/plugin-terser';
```

Comment:
do we still need rollup if we use tshy?

Reply:
Yes, to ship the umd bundled result. I wonder if we need to keep shipping it tho.